azavea/raster-foundry
# Add Dropbox credential field

## Overview

This will allow us to store a user's Dropbox credentials and use them to export rasters to that user's Dropbox account.

### Checklist


## Testing Instructions

 * Run migrations
 * Figure out your user ID somehow (plenty of requests in the UI have it)
 * Open up a `./scripts/psql` and then `update users set dropbox_credential='12345' where id=[whatever]`
 * Use a REST client to visit `/api/users/me/` and verify that you receive the the credential string that you entered previously.

Closes #1648
